src/data/pokemon/species_info/gen_1.h

```diff
@@ -10188,6 +10182,12 @@ const struct SpeciesInfo gSpeciesInfoGen1[] =
 
     [SPECIES_TAUROS] =
     {
+        .baseHP        = 75,
+        .baseAttack    = 100,
+        .baseDefense   = 95,
+        .baseSpeed     = 110,
+        .baseSpAttack  = 40,
+        .baseSpDefense = 70,
         TAUROS_MISC_INFO,
```

Comment:
`TAUROS_MISC_INFO` should precede all else, if I'm not mistaken

Reply:
fixed